src/index/repositories.props

```diff
@@ -52,40 +52,6 @@
         build
       </PrepareCommand>
     </Repository>
-    <Repository Include="corefxlab">
-      <Url>https://github.com/dotnet/corefxlab</Url>
-      <Projects>
-        src/**/*.csproj;
-      </Projects>
-      <PrepareCommand>
-        build -SkipTests "true"
-      </PrepareCommand>
-    </Repository>
-    <Repository Include="iot">
-      <Url>https://github.com/dotnet/iot</Url>
-      <Projects>
-        src/**/*.csproj;
-      </Projects>
-      <PrepareCommand>
-        build
-      </PrepareCommand>
-    </Repository>
-    <Repository Include="msbuild">
-      <Url>https://github.com/Microsoft/msbuild</Url>
-      <Branch>master</Branch>
-      <DeepClone>true</DeepClone>
-      <Projects>
-        src/**/*.csproj;
-      </Projects>
-      <ExcludeProjects>
-        src/Deprecated/**/*.csproj;
-        src/*UnitTests*/**/*.csproj;
-        src/Samples/**/*.csproj;
-      </ExcludeProjects>
-      <PrepareCommand>
-        powershell -NoLogo -NoProfile -ExecutionPolicy Bypass -Command "eng/common/build.ps1 -restore"
-      </PrepareCommand>
-    </Repository>
     <Repository Include="machinelearning">
```

Comment:
I think this one is still using 3.1 as well.

https://github.com/dotnet/machinelearning/blob/master/DotnetCLIVersion.txt

I’m not sure if it will cause problems or not.